website/docs/docs/cloud/connect-data-platform/connect-snowflake.md

```diff
@@ -48,6 +48,8 @@ to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowfl
 
 dbt Cloud supports Snowflake's [multi-factor authentication (MFA)](https://docs.snowflake.com/en/user-guide/security-mfa) as another username and password option for increased login security. Snowflake's MFA support is powered by the Duo Security service.
 
+Snowflake's MFA is available on all [plan types](https://www.getdbt.com/pricing).
```

Comment:
non blocker suggestion but maybe this can be turned into a callout and moved up to line 42 so it's more visible?